pkgs/development/python-modules/autofaiss/default.nix

```diff
@@ -38,9 +44,18 @@ buildPythonPackage rec {
     "pyarrow"
   ];
 
-  propagatedBuildInputs = [ embedding-reader fsspec numpy faiss fire pyarrow ];
+  propagatedBuildInputs = [
+    embedding-reader
+    fsspec
+    numpy
+    faiss
+    fire
```

Comment:
```
ERROR: Could not find a version that satisfies the requirement fire<0.5.0,>=0.4.0 (from autofaiss) (from versions: none)
ERROR: No matching distribution found for fire<0.5.0,>=0.4.0
```

Reply:
resolved in https://github.com/NixOS/nixpkgs/pull/207686/commits/5c39d7268ca4b706c97ee23b6ff0d15b0d0176e7